src/java.desktop/windows/native/libawt/windows/awt_Window.cpp

```diff
@@ -24,8 +24,8 @@
  */
 
 #include "awt.h"
-
 #include <jlong.h>
+#include <windows.h>
```

Comment:
Probably `windows.h` is not required… It compiled successfully without it, it should continue to compile now